### PR TITLE
Added cpp fibonacci series

### PR DESCRIPTION
**Pull request by akshaya-9**

### CODE REVIEW

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6329869&bid=20568200)

File: cpp_fibonacci.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+    int i, n, t1 = 0, t2 = 1, t3;
```
Comment (fharookshaik) [inline cpp_fibonacci.cpp:5]: Codacy found an issue: [The scope of the variable 't3' can be reduced.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6329869)

**Comment by fharookshaik** [comment]
Please check your code once.

**Review by akshaya-9 — COMMENTED** [review]
Output Image:

OutputFibo

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6329869&bid=20568200)

**Comment by fharookshaik** [comment]
I've made necessary changes to your Code. Thank you for your commitment and your contribution this this project. Star this project & share it if you find this interesting. 

Thank You!

Happy Coding!!
Happy Contributing!!

**Pull request merged**